backend/api/Controllers/Models/CreateInspectionAreaQuery.cs

```diff
@@ -1,9 +1,12 @@
-﻿namespace Api.Controllers.Models
+﻿using Api.Services;
+
+namespace Api.Controllers.Models
```

Comment:
Should perhaps the definition of InspectionAreaPolygon move to a models folder so we dont import services to models and models to services?

Reply:
Yeah, it should